python-packages/contract_wrappers/src/zero_ex/contract_wrappers/__init__.py

```diff
@@ -135,16 +135,21 @@
 ...     takerAssetAmount=to_wei(0.1, 'ether'),
 ...     expirationTimeSeconds=round(
 ...         (datetime.utcnow() + timedelta(days=1)).timestamp()
-...     )
+...     ),
+...     makerFeeAssetData='0x',
+...     takerFeeAssetData='0x',
+...     chain_id=Web3(ganache).eth.chainId,
```

Comment:
Is `chain_id` really assignable to an `Order`?  

When I run `mypy` on a file that looks like this:

```
order = Order(
    makerAddress='0x',
    takerAddress='0x0000000000000000000000000000000000000000',
    senderAddress='0x0000000000000000000000000000000000000000',
    feeRecipientAddress='0x0000000000000000000000000000000000000000',
    makerAssetData='0x',
    takerAssetData='0x',
    salt=1,
    makerFee=0,
    takerFee=0,
    makerAssetAmount=1,
    takerAssetAmount=1,
    expirationTimeSeconds=1,
    makerFeeAssetData='0x',
    takerFeeAssetData='0x',
    chain_id=42
)
```

I get the complaint: 
```
error: Extra key 'chain_id' for TypedDict "Order"
```

Note, this is using contract-wrappers `2.0.0.dev9`

Reply:
Thanks @steveklebanoff ! Fixed in https://github.com/0xProject/0x-monorepo/pull/2284/commits/19fac9206cf760566d67b13a1d6ea2eeed7ee3b8